src/databricks/labs/ucx/source_code/known.json

```diff
@@ -3141,6 +3141,83 @@
     "dockerpycreds.utils": [],
     "dockerpycreds.version": []
   },
+  "duckdb": {
+    "adbc_driver_duckdb": [],
+    "adbc_driver_duckdb.dbapi": [],
+    "duckdb": [
+      {
+        "code": "table-migrate",
+        "message": "Can't migrate 'conn.table(table_name, **kwargs)' because its table name argument cannot be computed"
+      },
+      {
+        "code": "table-migrate",
+        "message": "Can't migrate table_name argument in 'conn.sql(query, **kwargs)' because its value cannot be computed"
+      }
+    ],
+    "duckdb.bytes_io_wrapper": [],
+    "duckdb.experimental": [],
+    "duckdb.experimental.spark": [],
+    "duckdb.experimental.spark._globals": [],
+    "duckdb.experimental.spark._typing": [],
+    "duckdb.experimental.spark.conf": [],
+    "duckdb.experimental.spark.context": [],
+    "duckdb.experimental.spark.errors": [],
+    "duckdb.experimental.spark.errors.error_classes": [],
+    "duckdb.experimental.spark.errors.exceptions": [],
+    "duckdb.experimental.spark.errors.exceptions.base": [],
+    "duckdb.experimental.spark.errors.utils": [],
+    "duckdb.experimental.spark.exception": [],
+    "duckdb.experimental.spark.sql": [],
+    "duckdb.experimental.spark.sql._typing": [],
+    "duckdb.experimental.spark.sql.catalog": [
+      {
+        "code": "table-migrate",
+        "message": "Can't migrate table_name argument in 'self._session.conn.sql(query)' because its value cannot be computed"
```

Comment:
I expect this also to be a false positive, but should be verified more thoroughly due to the `spark` in the module name. DuckDB mimics the Spark API, still I expect this to be only duckdb

Reply:
fixed

Reply:
The `table-migrate` is still there